data.sql

```diff
@@ -1,35 +1,31 @@
 /* Populate database with sample data. */
+INSERT INTO
+  animals(
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES
+  ('Agumon', '2020-02-03', 0, TRUE, 10.23),
+  ('Gabumon', '2018-11-15', 2, TRUE, 8),
+  ('Pikachu', '2021-01-07', 1, false, 15.04),
+  ('Devimon', '2017-05-12', 5, TRUE, 11);
 
--- INSERT data
-
-INSERT INTO animals(
+INSERT INTO
+  animals(
     name,
     date_of_birth,
     escape_attempts,
     neutered,
     weight_kg
-) VALUES (
-    'Agumon',
-    '2020-02-03',
-    0,
-    true,
-    10.23
-), (
-    'Gabumon',
-    '2018-11-15',
-    2,
-    true,
-    8
-), (
-    'Pikachu',
-    '2021-01-07',
-    1,
-    false,
-    15.04
-), (
-    'Devimon',
-    '2017-05-12',
-    5,
-    true,
-    11
-);
+  )
+VALUES
+  ('Charmander', '2020-02-08', 0, false, 11),
+  ('Plantmon', '2021-11-15', 2, TRUE, 5.7),
+  ('Squirtle', '1993-04-02', 3, false, 12.13),
+  ('Angemon', '2005-06-12', 1, TRUE, 45),
+  ('Boarmon', '2005-06-07', 7, TRUE, 20.4),
+  ('Blossom', '1998-10-13', 3, TRUE, 17),
+  ('Ditto', '2022-05-14', 4, TRUE, 22);
```

Comment:
- [ ] Please make sure to insert the given data correctly. From the following screenshot, you can see some of the given data which are negative.

![image](https://github.com/DilsherB/vet-DB/assets/62290545/9a68c411-5657-4060-a392-bb3172ab162d)

Reply:
@ShadyShawkat please watch this video and tell me about the issue you raised.
https://youtu.be/m2fIWhlP2OQ